File: gen_scripts/php/g20.array.php
#!/usr/bin/env php
<?php
$nicknames = array("bob","ed","steve","ralph","joe","deb","kate");
echo "The names are: \n";
for ($count=0; $count < count($nicknames); $count++) {
    echo " nicknames[$count]=$nicknames[$count]\n";
}
?>

File: gen_scripts/python/g20.array.py
#!/usr/bin/env python
# create a populated list
nicknames = ["bob","ed","steve","ralph","joe","deb","kate"]

# output the list with index, one item per line
print "The names are: "
# utitlize count style loop to enumerate list
for count in range(len(nicknames)):
    print " nicknames[%d]=%s" % (count, nicknames[count])